index.html

```diff
@@ -14,24 +14,25 @@
 <body>
   <!-- HEADER STARTS HERE -->
   <header class="portfolio-header">
-    <nav class="navigation">
-      <div class="portfolio-logo">
-        <a href="#" class="portfolio-text">My Portfolio</a>
-      </div>
-      <div class="menu">
-        <ul class="menu-items">
-          <li class="menu-item"><a href="#">Portfolio</a></li>
-          <li class="menu-item"><a href="#">About</a></li>
-          <li class="menu-item"><a href="#">Contact</a></li>
+    <nav>
+      <div class="nav-center">
+        <div class="nav-header">
+          <div class="logo">
+            <h1>My Portfolio</h1>
+          </div>
+          <div class="nav-toggle">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </div>
```

Comment:
![Screenshot from 2022-11-15 01-09-20](https://user-images.githubusercontent.com/65068771/201788185-68caf200-f88e-4c11-8811-2065440dad06.png)

- [ ] Your project looks good and has a good design but you still have something to fix. :smile: 
- Initially, when the user clicks (or taps) the hamburger button everything is showing well but when the user clicks (or taps) the close (X) button, the hamburger button becomes hidden. :+1: